docs/src/tutorials/algorithms/tsp_lazy_constraints.jl

```diff
@@ -117,7 +117,7 @@ import Plots
 # thus, the weight (distance) of each edge is defined as follows.
 
 function generate_distance_matrix(n; random_seed = 1)
-    Random.seed!(random_seed)
+    Random.seed!(Random.MersenneTwister(1), random_seed)
     X = 100 * rand(n)
     Y = 100 * rand(n)
```

Comment:
Doesn't this need to be:
```julia
    rng = Random.MersenneTwister(random_seed)
    X = 100 * rand(rng, n)
    Y = 100 * rand(rng, n)
```
Otherwise I don't think it's reproducible.

Reply:
Yes, you're right.

Reply:
Not sure how I broke the format-check though.

Reply:
I'm fixing that in #2865. Not sure why either. Must be a transitive dependency change?

Reply:
Seems to be fixed.